tests/unit/sparql/buildQuery.spec.ts

```diff
@@ -162,10 +172,35 @@ describe( 'buildQuery', () => {
 				value,
 				datatype: 'string',
 				propertyValueRelation,
+				subclasses: false,
 			},
 		],
 		omitLabels: false,
 		} );
 		expect( actualQuery.replace( /\s+/g, ' ' ) ).toEqual( expectedQuery.replace( /\s+/g, ' ' ) );
 	} );
+
+	it( 'builds a query from a property and a string value with subclasses', () => {
+		const propertyId = 'P666';
+		const value = 'blah';
+		const subclassesId = 'P279';
```

Comment:
How do we pass the environment variable here? I tried `const subclassesId = process.env.VUE_APP_SUBCLASS_PROPERTY;` and it didnt work

Reply:
You could try 

```
process.env = Object.assign(process.env, { VUE_APP_SUBCLASS_PROPERTY: 'value' });
```

( which I found there: https://github.com/vuejs/vue-test-utils/issues/193#issuecomment-438437626 )

Reply:
Uf!! That was hard. The proposed solution wasn't working but this one worked: https://stackoverflow.com/a/58953365/1619792